sdk/honcho/__init__.py

```diff
@@ -1,2 +1,3 @@
-from .client import Client
+from .client import AsyncClient
+from .syncclient import Client
```

Comment:
Nitpick would prefer the name to be `sync_client` just to stick to snake case with python. I know it's technically a class but just not a fan of it for whatever reason. 

Honestly this file could be renamed to `core` with some of the being added elsewhere